sirepo/package_data/static/js/sirepo.js

```diff
@@ -812,6 +817,16 @@ SIREPO.app.factory('appState', function(errorService, fileManager, requestQueue,
         return model;
     };
 
+    self.superClasses = (modelName) => {
```

Comment:
Is there a reason this is classes plural? Looking through schemas I only see that things have one superClass.

Reply:
The ```_super``` array gets extended by ```simulation_db._merge_subclasses()``` so it contains the chain of supers back to the root